speech/cloud-client/beta_streaming.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Speech API sample application using the streaming API.
+
+Example usage:
+    python beta_streaming.py resources/audio.raw
+"""
+
+import argparse
+from collections import deque
+import io
+
+
+# [START speech_transcribe_diarization_streaming]
```

Comment:
I believe the new guideline suggests to have the region tags inside the function body (at least this is the case for nodejs).